Source/WebCore/rendering/style/RenderStyleConstants.h

```diff
@@ -103,17 +103,17 @@ enum class PseudoId : uint32_t {
     ViewTransitionNew,
 
     // Internal:
-    ScrollbarThumb,
-    ScrollbarButton,
-    ScrollbarTrack,
-    ScrollbarTrackPiece,
-    ScrollbarCorner,
-    Resizer,
+    WebKitScrollbarThumb,
```

Comment:
If these are really “internal” then maybe we should use a different kind of prefix?

Reply:
These are web exposed, so they're not really internal in the sense that `-internal-` is used for. I'll investigate this as a follow up to disambiguate the meaning of "Internal" here.